content/pt/docs/reference/tools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Ferramentas
+content_template: templates/concept
+---
+
+{{% capture overview %}}
+O Kubernetes contém várias ferramentas internas para ajudá-lo a trabalhar com o sistema Kubernetes.
+{{% /capture %}}
+
+{{% capture body %}}
+## Kubectl
+
+[`kubectl`](/docs/tasks/tools/install-kubectl/) é a ferramenta de linha de comando para o Kubernetes. Ela controla o gerenciador de cluster do Kubernetes.
+
+
+## Kubeadm
+
+[`kubeadm`](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) é a ferramenta de linha de comando para provisionar facilmente um cluster Kubernetes seguro sobre servidores físicos ou na nuvem ou em máquinas virtuais (atualmente em alfa).
+
+
```

Comment:
`Kubefed` is missing here

https://kubernetes.io/docs/reference/tools/#kubefed

Reply:
I decided not to add because according to #19876 and https://github.com/kubernetes/website/pull/19935 the Kubefed it is no longer maintained

Reply:
good catch!